app/controllers/open_library_controller.rb

```diff
@@ -0,0 +1,10 @@
+class OpenLibraryController < ApplicationController
+  def show
+    book = OpenLibrary.new(params[:isbn]).fetch_data
+    if book.empty?
+      render json: book, status: :unprocessable_entity
+    else
+      render json: book, status: :ok
```

Comment:
The status and response isn't right. When you can't find a book it's 404 resource not found.
You can read more about it here: https://github.com/Wolox/tech-guides/blob/master/backend/backend-standard-and-best-practices.md#32--error-forwarding

Basically the response should be: { error: { some error and message}, status: status_code }